src/components/SchemaInfoIcon.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+    <a v-bind:href="'https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md#' + section">
```

Comment:
```suggestion
    <a v-bind:href="'https://github.com/citation-file-format/citation-file-format/blob/1.2.0/schema-guide.md' + anchor">
```

- main -> 1.2.0 for more predictable results
- changed `section` to `anchor`, I believe that's what that is called (needs verification). Also I moved the `#` to be part of what components use, this makes it easier to recognize how that piece of data is used in my opinion